pkg/server/helpers.go

```diff
@@ -97,7 +97,7 @@ const (
 const (
 	// sandboxMetadataLabel is label name that identify metadata of sandbox in CreateContainerRequest
 	sandboxMetadataLabel = "io.cri-containerd.sandbox.metadata"
-	// sandboxMetadataLabel is label name that identify metadata of container in CreateContainerRequest
+	// containerMetadataLabel is label name that identify metadata of container in CreateContainerRequest
```

Comment:
it's not a label name any more, so perhaps "...is an extension name..."?

BTW The "sandboxMetadataLabel is label name" should really be "sandboxMetadataLabel is a label name".

Reply:
done. Thank for review